drivers/hud_data_quality_report/app/models/hud_data_quality_report/generators/fy2022/base.rb

```diff
@@ -90,7 +90,7 @@ class Base < ::HudReports::QuestionBase
           disabilities_latest = enrollment.disabilities.select { |d| d.InformationDate == max_disability_date }
 
           health_and_dv = enrollment.health_and_dvs.
-            select { |h| h.InformationDate <= report_end_date }.
+            select { |h| h.InformationDate <= report_end_date && !h.DomesticViolenceVictim.nil? }.
```

Comment:
what is this part doing?

Reply:
That's a back port from the APR that was missed at some point.  Basically, your health and dv records are only included if the DV question was answered (true or false, but not empty.)